debug-cache-hits.mjs

```
@@ -0,0 +1,112 @@
+try {
+  // Get current stats
+  const stats = await cache.getStats()
+  console.log('📊 Current Cache Stats:')
+  console.log(
+    '   Request Count:',
+    stats.hitRate === 0
+      ? 'Unknown (hitRate = 0)'
+      : Math.round(cache.hitCount / stats.hitRate),
+  )
+  console.log('   Hit Count:', 'Unknown (private property)')
+  console.log('   Hit Rate:', (stats.hitRate * 100).toFixed(1) + '%')
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)